dbms/programs/client/Client.cpp

```diff
@@ -130,7 +131,7 @@ class Client : public Poco::Util::Application
     bool print_time_to_stderr = false;   /// Output execution time to stderr in batch mode.
     bool stdin_is_not_tty = false;       /// stdin is not a terminal.
 
-    winsize terminal_size {};            /// Terminal size is needed to render progress bar.
+    unsigned short int terminal_width;   /// Terminal width is needed to render progress bar.
```

Comment:
Bug.

You have missed zero-initialization.

Reply:
I actually made function getTerminalSize() return 0 when no terminal detected, it should be moved from if case to be always initialized. Will fix as soon as possible, or maybe this is not a worse sollution.

Reply:
I just preserved old behaviour.

`s{}` for POD means zero initialization.